sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -593,6 +595,34 @@ public GlobalThroughputControlConfigBuilder createGlobalThroughputControlConfigB
         return new GlobalThroughputControlConfigBuilder(this, databaseId, containerId);
     }
 
+    /**
+     * Sets the regions to exclude from the list of preferred regions. This means the request will not be
+     * routed to these excluded regions for non-retry and retry scenarios
+     * for the workload executed through this instance of {@link CosmosAsyncClient}.
+     *
+     * @param excludedRegions the list of regions to exclude.
+     * */
+    public void setExcludedRegions(List<String> excludedRegions) {
```

Comment:
Should this be a `Set` instead? - how do we handle duplicates currently?

Reply:
It is mentioned in the PR description that the mutability only works if the same API is called on `CosmosClientBuilder` first. But we are not providing any documentation here. 
What happens if customer directly calls this API without calling the client builder API first?

Reply:
I checked the preferredRegions API as well - that is also a List. 
So I think its good to keep the consistency, but we should handle duplicates well in that case.

Reply:
Oh, the mutability should be irrespective of the source of injection - `CosmosClientBuilder` or `CosmosClient`. Internally - we do handle duplicates since excludedRegions acts as a filter to preferredRegions but since I am also exposing a getter for excludedRegions, filtering out the duplicates explicitly.

Reply:
Duplicate handling added in next iteration.